### PR TITLE
Sign-up Page

### PR DESCRIPTION
**Pull request by phatwillis**
# Sign-up Page and #294 

Title : Sign-up Page

Issue No. : #294 

Code Stack : Html, Css, JavaScript.

Close #<294>

# Checklist:

- [X] I have mentioned the issue number in my Pull Request.
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

### CODE REVIEW

**Comment by arpansaha13** [comment]
You need to add the challenge link in `challenges.ts` for it to appear in the UI.

You can read about it in the [Contributing guide](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/CONTRIBUTING.md#steps-to-contribute-javascript-challenge).

**Review by arpansaha13 — CHANGES_REQUESTED** [review]
You need to add the challenge link in `challenges.ts` for it to appear in the UI.

You can read about it in the [Contributing guide](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/CONTRIBUTING.md#steps-to-contribute-javascript-challenge).

File: apps/javascript/src/challenges/signup/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            <input type="text" id="email" name="email"><br>
+            <label for="full-name">Full Name:</label><br>
+            <input type="text" id="full-name" name="full-name">
+            <label for="dob">DOB:</label><br>
+            <input type="date" id="dob" name="dob"><br>
+            <label for="pass">Password:</label><br>
+            <input type="text" id="pass" name="passs">
+            <input type="submit" value="Submit">
+        </form>
+        <script src="script.js" type="module"></script>
```
Comment (sadanandpai) [inline apps/javascript/src/challenges/signup/index.html:21]: move it to head

File: apps/javascript/src/challenges/signup/style.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
+body {
+    font-family: 'Arial', sans-serif;
```
Comment (sadanandpai) [inline apps/javascript/src/challenges/signup/style.css:2]: not needed

File: apps/javascript/src/challenges/signup/style.css
Comment (sadanandpai) [inline apps/javascript/src/challenges/signup/style.css:1]: import core styles

File: apps/javascript/src/challenges/signup/script.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// An event listener to the form for submission
+document.querySelector('form').addEventListener('submit', function (event) {
+    event.preventDefault(); // Prevent the default form submission
+
+    // Retrieve form values
+    const email = document.getElementById('email').value;
+    const fullName = document.getElementById('full-name').value;
+    const dob = document.getElementById('dob').value;
+    const password = document.getElementById('pass').value;
```
Comment (sadanandpai) [inline apps/javascript/src/challenges/signup/script.js:9]: can we keep it outside

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by sadanandpai** [comment]
is the challenge added to challenges list? @phatwillis

**Pull request closed without merging**